src/test/scala/chiselTests/VectorPacketIO.scala

```diff
@@ -40,6 +41,8 @@ class BrokenVectorPacketModule extends Module {
   val n  = 4
   val io = IO(new VectorPacketIO(n))
 
+  io := DontCare
```

Comment:
Why is this necessary?

Reply:
This suppresses the "Reference io is not fully initialized" error from firrtl. It seemed the minimum change to keep the test working.

Reply:
Can we assign something to it so this doesn't also test DontCare behavior?